packages/orchestration/README.md

```diff
@@ -158,11 +177,21 @@ try {
 }
 ```
 
+In the above code snippet, both `orchestrationClient.chatCompletion()` and `response.getContent()` can throw errors.
+
+- **Axios Errors**:  
+  When the chat completion request fails with a `400` status code, the caught error will be an `Axios` error. The property `error.response.data.message` may provide additional details about the failure's cause.
+
+- **Output Content Filtered**:  
+  The method `response.getContent()` can throw an error if the content filter configuration filters the output. This situation can occur even if the initial request succeeds. The `error.message` property indicates whether the output was filtered.
```

Comment:
```suggestion
  The `response.getContent()` method can throw an error if the output content filter configuration filters the output. This can occur even if the initial request succeeds. The `error.message` property indicates whether the output was filtered.
```

1. Maybe have a section for error handling? Currently this is inside content-filtering. Are axios errors limited to content filtering ?
2. What is the initial request?
3. Its not the configuration that filters the output, but the output filter.

Reply:
1. As currently,  none of the clients talk about error handling, I would rather open a follow-up task that adds error handling sections to every client. I personally don't have much understanding to do proper error handling in TS, so wouldn't be able to explain it in this document now.
Axios errors are not limited to content filtering, but I hope it is clear when Axios errors occur during content filtering.
2 & 3, I have worked upon.